crates/bindings-typescript/src/sdk/websocket_test_adapter.ts

```diff
@@ -3,6 +3,7 @@ import { ServerMessage } from './client_api/index.ts';
 
 class WebsocketTestAdapter {
   onclose: any;
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
   onopen!: Function;
   onmessage: any;
   onerror: any;
```

Comment:
```suggestion
  onopen!: () => void;
  onmessage: (message) => void;
  onerror: (error) => void;
```

Why not something like this?

Reply:
I was trying not to change code that is currently working. I just added the eslint ignore line because this part of the code was not previously linted. I thought it was less risky to silence the warning than to update the code simply because I didn't understand what the intention of the author was.

Reply:
Fair enough!